src/sync/impl/sync_file.cpp

```diff
@@ -83,35 +83,6 @@ char decoded_char_for(const std::string& percent_encoding, size_t index)
 
 namespace util {
 
-void remove_nonempty_dir(const std::string& path)
```

Comment:
There should be a corresponding removal of this from sync_file.hpp. The two callers in our tests will need to be updated to use `util::try_remove_dir_recursive` as well.

Reply:
Whoops, missed that this wasn't a private helper function.